diagnostic.md

```diff
@@ -6,12 +6,14 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    hiding and showing div ?
```

Comment:
Can you expand on this more?  Common UI examples would be a list w/ list-items, a navbar w/ buttons, a sidebar for a blog w/ comments.